From fe5738d76ff440a5373759f1895d4a85f36041b0 Mon Sep 17 00:00:00 2001
From: Nolen Johnson <johnsonnolen@gmail.com>
Date: Mon, 17 Sep 2018 23:08:46 +0000
Subject: [PATCH] Updater: Allow Cleartext HTTP communications

* As of 9.0, Android disallows ClearText traffic over
  HTTP by default. Re-Enable it for those of us using
  an Updater instance that can't use SSL (i.e. behind
  a proxy, etc.).

Change-Id: I76f96e6424ce26922e9e196a0e1243b8494dc084
---
 AndroidManifest.xml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index e39b67c..a46a981 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -17,7 +17,8 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        android:usesCleartextTraffic="true" >
 
         <activity
             android:name=".UpdatesActivity"
-- 
2.17.1

